Routes/Product listing page/product.html

```diff
@@ -0,0 +1,343 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Shoe-kart | Home</title>
```

Comment:
You should interchange the order of the title here, the name of the current page should come before the brand name, because in smaller tabs only the first name would be visible and the name of the current page is more important for the user to know

Reply:
thanks for pointing it out. will change